lib/travis/build/script/csharp.rb

```diff
@@ -59,6 +68,24 @@ def script
             sh.cmd 'false', echo: false, timing: false
           end
         end
+
+        def mono_repo
+          if config[:mono] == 'latest'
+            'wheezy'
+          else
+            if MONO_VERSION_REGEXP === config[:mono]
+              'wheezy/snapshots/' << config[:mono]
+            else
+              'wheezy'
+            end
+          end
+        end
+
+        def mono_version_valid?
+          if MONO_VERSION_REGEXP === config[:mono] || config[:mono] == 'latest'
```

Comment:
nit: switch the two sides of the expression, "latest" should be the common case and it'll allow short-circuiting the check.